tests/contrib/gp/test_likelihoods.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import absolute_import, division, print_function
+
+from collections import namedtuple
+
+import pytest
+import torch
+
+from pyro.contrib.gp.kernels import RBF
+from pyro.contrib.gp.likelihoods import Binary, MultiClass
+from pyro.contrib.gp.models import SparseVariationalGP, VariationalGP
+
+T = namedtuple("TestGPLikelihood", ["model_class", "X", "y", "kernel", "likelihood"])
+
+X = torch.tensor([[1, 5, 3], [4, 3, 7], [3, 4, 6]])
+kernel = RBF(input_dim=3, variance=torch.tensor([1]), lengthscale=torch.tensor([3]))
```

Comment:
Can we make these scalars? `variance=torch.tensor(1.), lengthscale=torch.tensor(3.)`

Reply:
Yes, we can use scalars after HMC supports it :D